src/pvrclient-argustv.cpp

```diff
@@ -712,7 +714,9 @@ PVR_ERROR cPVRClientArgusTV::GetRecordings(ADDON_HANDLE handle)
               }
               PVR_STRCPY(tag.strTitle, recording.Title());
               PVR_STRCPY(tag.strPlotOutline, recording.SubTitle());
-              PVR_STRCPY(tag.strStreamURL, recording.RecordingFileName());
+
+              std::string strRecId = tag.strRecordingId;
+              m_RecordingsMap[strRecId] = recording.RecordingFileName();
```

Comment:
I think you can rewrite this to one line: `m_RecordingsMap[tag.strRecordingId] = recording.RecordingFileName();` The compiler should know how to create a std::string from the char[].